lua/chadrc.lua

```diff
@@ -126,6 +126,31 @@ local M = {
          toggle_linenr = "<leader>n",
          theme_toggle = "<leader>x",
       },
+      saga = {
+         lsp_finder = "<leader>af",
+         code_action = "<leader>aa",
+         hover_doc = "<leader>ah",
+         signature_help = "<leader>as",
+         rename = "<leader>ar",
+         preview_definition = "<leader>ad",
+         diagnostic_jump_next = "<leader>an",
+         diagnostic_jump_prev = "<leader>dp",
+         open_floatterm = "<leader>ao",
+         close_floatterm = "<leader>ac",
+
+      },
+      hop = {
+        hop_line = "<leader>pl",
+        hop_word = "<leader>pw",
+        hop_char1 = "<leader>p1",
+        hop_char2 = "<leader>p2",
+
+      },
+      trouble = {
```

Comment:
I think Trouble provides some potentially nice functionality,
though I do think that this the user would have their own opinions as to if they want it or not.

Definitely something that should have an `enable/disable` option in `chadrc.lua`, and we should move to prompting what the user wants enabled/disabled on `install.sh`